### PR TITLE
Use github release versioning scheme

### PR DESCRIPTION
**Pull request by buildbreaker**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ jobs:
           distribution: temurin
           java-version: 11
           check-latest: true
+      - name: Set release version
+        run: |
+          release_version=$(git describe --tags --abbrev=0 --exact-match)
+          export ORG_GRADLE_PROJECT_releaseVersion="${release_version:1}"
```
Comment (andrewparmet) [inline .github/workflows/release.yml:31]: Is the `:1` a default? Would that cause issues (i.e., a bad publication) if it was used?

Comment (buildbreaker) [reply .github/workflows/release.yml:31]: I cut off the first char cause it's always `v`. The convention for publishing is to do `v*` but we strip the v in the publish version

Comment (andrewparmet) [reply .github/workflows/release.yml:31]: First char of what? I'm asking specifically about the `:1` that is in quotes. What does that do?

Comment (andrewparmet) [reply .github/workflows/release.yml:31]: Ah, substring? https://stackoverflow.com/a/15143895

**Review by andrewparmet — APPROVED** [review]

**Pull request merged**